code/modules/power/fission/core_control.dm

```diff
@@ -0,0 +1,118 @@
+/obj/machinery/computer/fission
+	name = "fission control computer"
+	icon_keyboard = "power_key"
+	icon_screen = "fission_screen"
+	light_color = COLOR_ORANGE
+	idle_power_usage = 250
+	active_power_usage = 500
+	var/initial_id_tag = "fission"
+	
+	var/current_core = 1 // Which core connected to the console is being controlled/viewed.
+	var/current_rod = 0  // Which fuel rod in the selected core is being viewed. 0 if no rod is being viewed.
+	var/diagnostics = FALSE // Whether or not the control computer displays advanced diagnostics.
+
+/obj/machinery/computer/fission/Initialize()
+	set_extension(src, /datum/extension/local_network_member)
+	if(initial_id_tag)
+		var/datum/extension/local_network_member/fission = get_extension(src, /datum/extension/local_network_member)
+		fission.set_tag(null, initial_id_tag)
+	
+	. = ..()
```

Comment:
You probably want to initialize before you start messing with extensions. Just a vague feeling.